src/data/mod.rs

```diff
@@ -500,3 +508,27 @@ fn handle_readout_gpu(readout_values: &mut Vec<Readout>, general_readout: &Gener
         Err(e) => readout_values.push(Readout::new_err(ReadoutKey::GPU, e)),
     };
 }
+
+fn handle_readout_disk_space(
+    readout_values: &mut Vec<Readout>,
+    general_readout: &GeneralReadout,
+    theme: &Theme,
+    opt: &Opt,
+) {
+    use crate::format::disk_space as format_disk_space;
+
+    match general_readout.disk_space() {
```

Comment:
I agree but this will need changes to libmacchina as I believe currently it's hard coded to "/", I can get to this later but I won't be free for about 1.5 weeks unfortunately